README.md

```diff
@@ -63,6 +63,14 @@ To communicate with the Hetzner Cloud API, you need to expose a `HCLOUD_TOKEN` e
 $ export HCLOUD_TOKEN="set_the_hcloud_token_here"
 ```
 
+You furthermore can set a different type for the ssh key used by molecule to connect to the Hetzner Cloud server. The default is set to rsa. To use different types export the MOLECULE_SSH_KEY_TYPE environment variable.
+
+```bash
+$ export MOLECULE_SSH_KEY_TYPE="ed25519"
```

Comment:
The MOLECULE_ prefix is reserved by molecule, I'd rather use another one, or use a different method to configure this value.